lib/screens/test_screen/test_screen.dart

```diff
@@ -1,18 +1,119 @@
+import 'package:flutter_svg/parser.dart';
+import 'package:flutter_svg/svg.dart';
+
 import '../../general_exports.dart';
+import '../../generated/assets.dart';
 
 class TestScreen extends StatelessWidget {
   const TestScreen({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: SingleChildScrollView(
-        child: Center(
-          child: CommonText(
-            text: 'my_profile'.tr,
+      body: SafeArea(
+        child: SingleChildScrollView(
+          child: Center(
+            child: Column(
+              children: <Widget>[
+                const SizedBox(height: 20),
+
+                // TODO: the color used to test
+                commonButton(
+                    title: argTextButton,
+                    width: 0.86,
+                    height: 0.064,
+                    isIcon: true,
+                    backgroundColor: AppColors.green,
+                    fontSize: 18,
+                    borderColor: AppColors.black,
+                    fontWeight: FontWeight.w600,
+                    path: Assets.iconsIcEmail),
+                const SizedBox(
+                  height: 10,
+                ),
+                commonButton(
+                    title: argTextButton,
+                    width: 0.86,
+                    height: 0.064,
+                    isIcon: false,
+                    backgroundColor: AppColors.green,
+                    fontSize: 18,
+                    borderColor: AppColors.black,
+                    fontWeight: FontWeight.w600,),
+              ],
+            ),
           ),
         ),
       ),
     );
   }
+
+  /// This function take more parameter
+  /// @author omar ezzdeen
+  /// @return CommonButton Widget
+  ///
+  /// @param1 title this param is required
+  /// @param2 width this param is required
+  /// @param3 height this param is required
+  /// @param4 isIcon this param is optional
+  /// @param5 backgroundColor this param is optional
+  /// @param6 borderColor this param is optional
+  /// @param7 fontSize this param is optional
+  /// @param8 fontWeight this param is optional
+  /// @param9 path this param is optional
+  CommonButton commonButton({
```

Comment:
The name of the file should be capital letter and move it to a separated folder in components folder